schemas/Error.json

```
@@ -15,17 +15,18 @@
     },
     "name": {
       "type": "string",
-      "enum": [
-        "UnknownError",
-        "OlmIndexError",
-        "OlmKeysNotSentError",
-        "OlmUnspecifiedError",
-        "VoipUserHangup",
-        "VoipIceFailed",
-        "VoipInviteTimeout",
-        "VoipIceTimeout",
-        "VoipUserMediaFailed",
-        "ToDeviceFailedToDecrypt"
+      "oneOf": [
+        {"const": "OlmIndexError", "description": "E2E domain error. The room key is known but is ratcheted (index > 0)."},
+        {"const": "OlmKeysNotSentError", "description": "E2E domain error. Generic unknown inbound group session error."},
+        {"const": "OlmUnspecifiedError", "description": "E2E domain error. Any other decryption error (missing field, format errors...)."},
+        {"const": "UnknownError", "description": "E2E domain error. Decryption fails due to unknown error."},
+        {"const": "HistoricalMessage", "description": "E2E domain error. Decryption fails for historical message and there is no backup."},
```

Can we define what "historical message" means a bit better?

```suggestion
        {"const": "HistoricalMessage", "description": "E2E domain error. Decryption failed for a message sent before the device logged in, and key backup is not enabled."},
```